### DISCUSSION TITLE
A way to export multiple stories at once in an array, rather than named exports?

### DISCUSSION THREAD

**Discussion by rwacarter** [storybookjs/storybook#21651] [Help]
So the typical way to export stories from a file is named exports:
```
export const Basic = Template.bind({});
export const Another = Template.bind({});
```
My use case is that I would like to be able to generate the stories in another file, and export them from the stories file. Something like this:
```
export const stories = getStoriesForTemplate(Template);
```
Where `getStoriesForTemplate` is a function that will manipulate the Template is various ways, and return an array of stories.

Obviously that example doesn't work, and I can't figure out a way of being able to use named exports while intentionally not knowing what `getStoriesForTemplate` is returning. I know something like this would work:
```
export const [Basic, Another] = getStoriesForTemplate(Template);
```
But I would like to try and avoid having to define the stories in this file if possible, so they can change etc without needed to update this file.

**Comment by nstuyvesant** [comment]
This is how we are doing it now...
https://github.com/storybookjs/storybook/discussions/21559

It works with Storybook 7 (somewhat) but we had to add this to the config in .storybook/main.ts...
```js
features: {
  storyStoreV7: false
}
```
And we get warnings that we shouldn't use storiesOf() any longer and when we run Storybook, we get a message that the components are not configured for Controls.
